handler/handler.go

```diff
@@ -35,7 +35,10 @@ func (h *Handler) write(w http.ResponseWriter, status int, body []byte) {
 
 func (h *Handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	resp, err := h.ProxyClient.Do(r)
-	if err != nil {
+	if r.URL.Path == "/health" {
```

Comment:
Since the main goal of sigv4-proxy is to proxy a request, it should not assuming/mingle with the received URL path. Assuming that when `URL.Path = /health` then it mean health checking this proxy is ambiguous; does the caller meant to health check the AWS service or this proxy? 

I understand you are trying to address https://github.com/awslabs/aws-sigv4-proxy/issues/193, but having ALB checking sigv4-proxy's health is not generally useful, what is more useful is for sigv4-proxy to forward the health check request to underlying service.